Makefile

```diff
@@ -155,11 +152,6 @@ COMMON_LDFLAGS += `pkg-config tinfo 2>/dev/null && pkg-config tinfo --libs || ec
 # TODO should probably do the linking before MCJIT
 COMMON_LDFLAGS += -Wl,-E
 
-# We get multiple shared libraries (libstdc++, libgcc_s) from the gcc installation:
```

Comment:
I think instead of removing parts of the unused makefile code in this commit we should leave it in and instead remove the whole non cmake related build code from the makefile sometime in the future.
If it really does not work without removing `ifneq ($(GCC_DIR),/usr)` than I guess there is more stuff used than I thought I will have review it more carefully again.